tests/exporters/test_tentacle_bundle_exporter.py

```diff
@@ -112,3 +113,21 @@ async def test_tentacle_bundle_exporter_with_specified_output_dir(install_tentac
     assert "daily_trading_mode.zip" in output_files
     assert "generic_exchange_importer@1.2.0" in output_files
     shutil.rmtree(specified_output_dir)
+
+
+async def test_tentacle_bundle_exporter_with_metadata_injection(install_tentacles):
+    assert await create_tentacles_package(package_name=TENTACLE_PACKAGE,
+                                          output_dir=constants.CURRENT_DIR_PATH,
+                                          metadata_file=os.path.join("tests", "static", "metadata.yml"),
+                                          in_zip=True,
+                                          use_package_as_file_name=True) == 0
+    assert os.path.exists(constants.ARTIFACT_METADATA_FILE)
```

Comment:
We should use `os.path.join("tests", "static", "metadata.yml")` instead of `constants.ARTIFACT_METADATA_FILE` for clarity I think